concent_api/core/migrations/0005_auto_20180122_0044.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.8 on 2018-01-22 00:44
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0004_auto_20180119_1055'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='message',
+            name='task_id',
+            field=models.CharField(max_length=255, null=True),
+        ),
```

Comment:
This change requires a data migration to preserve existing data.

Reply:
It doesn't, int will be casted to string without problems.

Reply:
Interesting. So PostgreSQL can do int <-> string conversion in simple cases. That's cool.

Reply:
Int => string yes
String => int probably will ask about custom handling while migrating, if you don't provide any and there will be non-numeric string then it will fail.